### PR TITLE
fix password lookup's  use of f=v settings (#76551)

### PR DESCRIPTION
**Pull request by bcoca**
update tests

(cherry picked from commit 5d253a13807e884b7ce0b6b57a963a45e2f0322c)

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

lookup/password

### CODE REVIEW

File: test/units/plugins/lookup/test_password.py [hunk 1]
```diff
@@ -390,23 +403,30 @@ def test_content_written(self):
...
+        self.os_close = os.close
+        os.close = lambda fd: None
+        self.os_mkdir = os.mkdir
+        os.mkdir = lambda fd, mode: None
+        self.os_makedirs = os.makedirs
+        os.makedirs = lambda fd, mode: None
+        self.os_remove = os.remove
+        os.remove = lambda path: None
+        self.makedirs_safe = upath.makedirs_safe
+        upath.makedirs_safe = lambda path, mode: None
```
Comment (s-hertel) [inline test/units/plugins/lookup/test_password.py]: why backport all of the unnecessary changes in this file as far back as 2.13?

Comment (bcoca) [reply test/units/plugins/lookup/test_password.py]: Just a habit, I normally avoid making changes during backports if i can, to minimize introducing  more unknowns
